articles/what-are-rich-text-editors.mdx

```diff
@@ -0,0 +1,52 @@
+---
+_schema: default
+_uuid: 7630a593-3a92-49c0-8557-627ddd5112fe
+_created_at: 2024-06-12T16:46:11+12:00
+title: What are rich text editors?
+nav_title: What are rich text editors?
+published: true
+image: >-
+  https://cc-dam.imgix.net/documentation/images/default-thumbnail/2023-Q4/CloudCannon-Documentation-Thumbnail.png
+description: Learn about rich text editors in CloudCannon.
+article_category:
+  - Explanation
+tags: []
+related_articles: []
+related_links: []
+explicit_canonical:
+---
+A rich text editor is an interface for editing content on your Site, with tools to format and style your content. Rich text editors use a WYSIWYG editing area, meaning that the way you format the content in the editor will resemble the way it will look on your web page (WYSIWYG is an acronym for What-You-See-Is-What-You-Get).
+
```

Comment:
Somewhere at the top here we should also mention the formats that are supported (markdown, mdx, html)